e2e/cypress/support/api/user.d.ts

```diff
@@ -340,5 +356,15 @@ declare namespace Cypress {
          *   cy.apiUpdateUserAuth('user-id', 'auth-data', 'password', 'auth-service');
          */
         apiUpdateUserAuth(userId: string, authData: string, password: string, authService: string): Chainable<Response>;
+
+        /**
+         * Update a user's auth data
+         * See https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1mfa/put
+         * @param {String} userId - ID of user to patch
+         * @param {String} authData - the new auth data to set
+         * @example
+         *   cy.apiUpdateAuthData('user-id', {auth_data: 'newdata', auth_service: 'saml'});
+         */
+        apiUpdateAuthData(userId: string, authData: string): Chainable<Response>;
```

Comment:
Btw, can we use/modify the existing `apiUpdateUserAuth` command since it's referring to the same REST API?